.github/workflows/goreleaser.yml

```diff
@@ -16,11 +16,11 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: "1.20"
+          go-version: "1.20.x"
           check-latest: true
           cache: true
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v4
+        uses: goreleaser/goreleaser-action@v3
```

Comment:
There is a breaking change in goreleaser going from `v3` to `v4`, I don't have the cycles to debug it atm.

```
  • loading config file                              file=.goreleaser.yml
  • checking config...
    • DEPRECATED: `archives.replacements` should not be used anymore, check https://goreleaser.com/deprecations#archivesreplacements for more info
  ⨯ command failed                                   error=config is valid, but uses deprecated properties, check logs above for details
```